client/src/Components/Admin/Pages/Admin.jsx

```diff
@@ -16,7 +16,7 @@ import{
 import './Admin.css';
 //Components
 import Dashboard from './Dashboard/Dashboard';
-import Users from './Users/Users'
+import Users from './users/Users'
```

Comment:
This bug! Not sure why git keeps lowercasing the directory 'users' but not the others. As soon as this is merged, there will be compile errors... Just fix by capitalizing this folder name!